react/PageBlock/PageBlock.less

```diff
@@ -0,0 +1,11 @@
+@import (reference) "~seek-style-guide/theme";
+
+.content {
+  max-width: @grid-container-width + (@grid-gutter-width * 2);
```

Comment:
@mjt01 since you last reviewed this, we noticed that this was incorrectly setting the page width at `940px`, when the page actually needs to be `980px` including gutters.